src/test/K6/src/wrapper.js

```diff
@@ -0,0 +1,25 @@
+import http from 'k6/http';
+
+//wrapper functions around k6 http methods enabling retrying requests when response code is 0, 408 and > 500
+
+export function httpPost(url, body, params) {
+  var res;
+  for (var retries = 3; retries > 0; retries--) {
+    res = http.post(url, body, params);
+    if (res.status != 0 && res.status != 408 && res.status < 500) {
+      return res;
+    }
```

Comment:
Should we wait 10 seconds before re-trying?

Reply:
ya, i can add a wait of 10 seconds before retrying